frontend/routes.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * React Navigation Routes
+ *
+ * Add type definitions and parameters for routes in {@link StackNavigator}
+ */
+export type Routes = {
+  /**
+   * Displays information about a culture, looking up information about that
+   * culture by provided prop {@link cultureName}.
+   *
+   * For admins it allows editing and adding information about a particular culture.
+   */
+  Culture: { cultureName: string };
+  Home: undefined;
+  Register: { token: string };
```

Comment:
Not sure if this is what you want, this would require that the prop be passed.

Do you want
```suggestion
  Register: { token?: string };
```

Meaning it can be undefined.

Reply:
Well, the user will need that token in order to register. So it is required, right?